source/synthDrivers/espeak.py

```diff
@@ -340,9 +338,15 @@ def speak(self, speechSequence: SpeechSequence):  # noqa: C901
 				langChanged = True
 			elif isinstance(item, BreakCommand):
 				textList.append(f'<break time="{item.time}ms" />')
+			elif isinstance(item, RateCommand):
+				if item.multiplier == 1:
+					textList.append("<prosody/>")
+				else:
+					textList.append(f"<prosody rate={int(item.multiplier * 100)}%>")
```

Comment:
can you see if this hack is still needed? 
@NSoiffer - do you have any insights on this?
RateCommand shoukd be supported below in a similar manner in `type(item) in self.PROSODY_ATTRS`.
The only difference is adding `textList.append("<prosody/>")` whenever `item.multiplier == 1`.
I think we should fix up / integrate the fix into the below code if it's necessary, rather than just for the ratecommand separately